src/main/java/com/powsybl/openloadflow/network/AbstractLfBranch.java

```diff
@@ -193,6 +186,19 @@ protected void checkTargetDeadband(RatioTapChanger rtc) {
         }
     }
 
+    protected static double getScaleForLimitType(LimitType type, LfBus bus) {
+        switch (type) {
+            case ACTIVE_POWER:
+            case APPARENT_POWER:
+                return 1.0 / PerUnit.SB;
+            case CURRENT:
+                return bus.getNominalV() / PerUnit.SB;
+            case VOLTAGE:
+            default:
+                throw new UnsupportedOperationException(String.format("Getting %s limits is not supported on dangling line branch.", type));
```

Comment:
Why this message? What is the link with dangling line?

Reply:
Probably copy pasted the exception throwing too fast.